features/src/main/feature/camel-features.xml

```diff
@@ -828,12 +828,18 @@
     <feature name='camel-elasticsearch' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <feature prerequisite='true'>wrap</feature>
+        <feature prerequisite="true">spifly</feature>
         <feature version='[2.16,2.17)'>jackson</feature>
         <feature version="[4,5)">http-client</feature>
+        <bundle dependency="true">mvn:org.glassfish.hk2/osgi-resource-locator/2.5.0-b42</bundle>
+        <bundle dependency="true">mvn:org.eclipse.parsson/parsson/${parson-version}</bundle>
+        <bundle dependency='true'>mvn:jakarta.json/jakarta.json-api/${jakarta-json-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:io.opentelemetry/opentelemetry-api/${opentelemetry-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:io.opentelemetry/opentelemetry-context/${opentelemetry-version}</bundle>
         <bundle dependency='true'>wrap:mvn:org.apache.httpcomponents/httpasyncclient/${httpasyncclient-version}</bundle>
-        <bundle dependency='true'>wrap:mvn:co.elastic.clients/elasticsearch-java/8.12.1</bundle>
-        <bundle dependency='true'>wrap:mvn:org.elasticsearch.client/elasticsearch-rest-client/8.12.1</bundle>
-        <bundle dependency='true'>wrap:mvn:org.elasticsearch.client/elasticsearch-rest-client-sniffer/8.12.1</bundle>
+        <bundle dependency='true'>wrap:mvn:co.elastic.clients/elasticsearch-java/${elasticsearch-java-client-version}</bundle>
```

Comment:
I would prefer to not mix unrelated changes: this is not related directly to the integration test.

Reply:
I agree, but we need real tests to validate the "framework" and while testing the camel ES feature, we realized that it doesn't work as it is. But no worries, if really needed, I will split it in 2 later today

Reply:
That's OK, just a note for next PRs ;)

Reply:
ok 👍